docs/CONTRIBUTING.md

```diff
@@ -75,10 +67,25 @@ Below is a list of other guidelines we try to follow for PRs.
   - As an exception to this rule, a reviewer who has just approved a PR can decide to rebase the branch via the GitHub UI if
     - the author has enabled auto-merge and
     - a branch update (without conflicts) is the only thing stopping the auto-merge
-- For substantial changes, we prefer creating a "parent/feature" PR which receives multiple smaller PRs. For the parent PR and all the smaller child PRs, all the general guidelines apply.
-- For PRs towards main
-  - Every PR should contain a reference to a devops task
+- PRs should have "the shortest life possible" and therefore be merged to `main` as quickly as possible, to avoid the overhead of maintenance. As a consequence:
+  - We make PRs as small as possible
+  - We prioritize reviewing code over writing new code
+  - We try to split larger changes/PRs into smaller chunks/refactors that can be merged independently
+  - We try to avoid mixing FE & BE changes in one PR
```

Comment:
This is vague.

Reply:
I added a clarification, let me know if it helps. If not, let me know what is unclear.

Reply:
I included your comment in the explanation.